src/packages/console/src/components/siteCard.tsx

```diff
@@ -37,30 +13,33 @@ interface ISiteCardProps {
 }
 
 export const SiteCard = (props: ISiteCardProps): React.ReactElement => {
-  const classes = useStyles();
 
   const onSiteClicked = (): void => {
     props.onSiteClicked(props.site);
   };
 
   return (
-    <Card className={props.isEnabled ? classes.card : classes.disabledCard}>
-      <CardActionArea className={classes.siteCardButtonBase} onClick={onSiteClicked} disabled={!props.isEnabled}>
-        <CardMedia
-          className={classes.media}
-          image={`https://${props.site.slug}.evrpg.com/screenshot-preview.png`}
-          title={props.site.name}
-        />
-        <CardContent className={classes.siteCardContent}>
-          <Typography variant='h6' className={classes.siteNameText}>
-            {props.site.name}
-          </Typography>
-          <Typography color='textSecondary'>
-            {props.site.slug}
-          </Typography>
-        </CardContent>
-      </CardActionArea>
-    </Card>
+    <BackgroundView color={!props.isEnabled && '#eee'}>
+      <Box variant='card-unpadded'>
+        <CardActionArea style={{ width: '100%' }} onClick={onSiteClicked} disabled={!props.isEnabled}>
```

Comment:
maybe LinkBase will work here?

Reply:
yeah, I was not sure what to use here, but I will try `LinkBase` here.